server/commands.py

```diff
@@ -125,12 +125,12 @@ def ooc_cmd_area_kick(client: ClientManager.Client, arg: str):
     /area_kick 3 Class Trial Room,\ 2   :: Kicks the player whose client ID is 1 to Class Trial Room, 2 (note the ,\).
     """
 
-    if not client.is_staff():
-        raise ClientError('You must be authorized to do that.')
+    Constants.assert_command(client, arg, is_staff=True, parameters='&1-2', split_spaces=True)
```

Comment:
This fails if the area name is more than 1 word long (e.g. /area 0 Class Trial Room)